spec/graphql/search_spec.rb

```diff
@@ -3,7 +3,7 @@
 describe GraphqlController, type: :controller do
   let(:jwt_token) do
     JWT.encode(
-      { aud: Rails.application.secrets.artsy_application_id },
+      { aud: Rails.application.secrets.artsy_application_id, roles: 'trusted' },
```

Comment:
I'd like to see another test case here without the trusted role that shows it doesn't work - are you good with adding that?

Reply:
Done